src/pages/Introduces/Introduce.scss

```
@@ -0,0 +1,38 @@
+@import "../../style/variables.scss";
+@import "../../style/mixin.scss";
+
+.introduce {
+  height: 100vh;
+  @include flexCenter;
```

```suggestion
  @include flexCenter;
  height: 100vh;
```

@include가 스타일 부분에 가장 위에 오는 것이 컨벤션입니다~